components/SocialMediaConnect.vue

```diff
@@ -219,18 +219,16 @@ export default {
     },
     ...mapGetters(['getUserInfo']),
   },
+  watch: {
+    platforms: {
+      handler(p) {
+        this.updateSelectedFacebookPage(p);
```

Comment:
since this function is explicitly named for facebook, the checking
```if (this.isLarge && platforms && platforms.facebook)``` should be done here, instead of inside the function?

Reply:
... or maybe rename this function to better reflect its usage? since I just realize it is also called in `mounted()`

Reply:
any suggestion? `refreshSelectedFacebookPage`?

Reply:
in `mounted()` and `watch` , sth like ```if (this.isLarge) this.refreshAddtionalDetails()```,
then inside `refreshAddtionalDetails()`
```if (this.platforms && this.platforms.facebook) 
 this.updateSelectedFacebookPage(this.platforms.facebook);```?

Reply:
Updated